assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
> In future not all devices will fit in the mobile,tablet,desktop categories. I used the exact width where the banner needs to be adjusted. This way any new device (say 977px) will adjust accordingly.

@roundrobin99 I don't really understand your reasoning please explain the exact width where this is needed and why. Drop a `.gif` in if that is easier to demonstrate why you need this padding at this exact width. 
Please be aware that this introduces random magic numbers into the codebase. Please use `641px` which equates to `tablet` in our codebase. This is what is used in `govuk-templates.css` https://github.com/hmrc/assets-frontend/blob/master/assets/govuk_elements/govuk/public/stylesheets/govuk-template.css#L315-L318